docs/configuration/uds-operator.md

```diff
@@ -191,6 +190,26 @@ spec:
 
 This configuration does not create a secret in the cluster and instead tells the UDS Operator to create a public client (one that requires no auth secret) that enables the `oauth2.device.authorization.grant.enabled` flow and disables the standard redirect auth flow.  Because this creates a public client configuration that deviates from this is limited - if your application requires both the Device Authorization Grant and the standard flow this is currently not supported without creating two separate clients.
 
+### SSO Client Attribute Validation
+
+```yaml
+apiVersion: uds.dev/v1alpha1
+kind: Package
+metadata:
+  name: fulcio
+  namespace: fulcio-system
+spec:
+  sso:
+    - name: Sigstore Login
+      clientId: sigstore
+      standardFlowEnabled: false
+      publicClient: true
+      attributes:
+        oauth2.device.authorization.grant.enabled: "true"
+```
+
+The [package-validator.ts](https://github.com/defenseunicorns/uds-core/blob/main/src/pepr/operator/crd/validators/package-validator.ts#L84) will verify that the attributes being used are allowed. If attributes are used that are not allowed the package will be denied and not installed.
+
```

Comment:
I don't know if this would become hard to maintain but should we just enumerate the supported attributes here? I think that might be more valuable than the example + link to code?

Reply:
```suggestion
The SSO spec supports a subset of the Keycloak attributes for clients, but does not support all of them. The current supported attributes are:
- oidc.ciba.grant.enabled
- backchannel.logout.session.required
- backchannel.logout.revoke.offline.tokens
- post.logout.redirect.uris
- oauth2.device.authorization.grant.enabled
- pkce.code.challenge.method
- client.session.idle.timeout

If you have a use-case for an additional attribute please open an issue for us to explore it.

```

Not sure on the last line there but this is more what I was thinking, quick note on supported attributes w/ the list.